content/docs/latest/installing/community-platforms/ovhcloud.md

```diff
@@ -0,0 +1,144 @@
+---
+title: Running Flatcar on OVH Cloud
+linktitle: Running on OVH Cloud
+weight: 10
+---
+
+These instructions will walk you through using Flatcar on [OVH Cloud][ovhcloud], importing an image, and running your first server using the OpenStack command line interface. Please note that OVH Cloud is a community supported platform at the moment which means that the CI does not run on OpenStack images in OVH Cloud environment.
+
+Additional note: OVH Cloud is compatible with OpenStack: it is not a mistake if we refer to OpenStack images and specific variables.
+
+## Import the image
+
+_Note_: This getting started assumes that OpenStack CLI is already installed and properly configured (see OVH Cloud [documentation][ovhcloud-documentation-import])
+
+It is possible to import your own Flatcar image in OVH Cloud using the OpenStack CLI. Here's example, to upload an image from the Flatcar Beta channel:
+
+```bash
+$ wget https://beta.release.flatcar-linux.net/amd64-usr/current/flatcar_production_openstack_image.img
+$ openstack image create \
+  --container-format bare \
+  --disk-format qcow2 \
+  --property image_original_user=core \
+  --property hw_qemu_guest_agent=yes \
+  --file flatcar_production_openstack_image.img \
+  flatcar-beta
+$ IMAGE_ID=$(openstack image show flatcar-beta --format json | jq -r .id)
```

Comment:
I don't have OpenStack on my PC here but can't we do this?
```suggestion
$ IMAGE_ID=$(openstack image show flatcar-beta --format value --column id)
```

Reply:
Actually you could just pass these to `openstack image create` and skip the `image show` command which could behave weirdly (fail?) if two images are named `flatcar-beta`.